src/cmd/cli/command/compose.go

```diff
@@ -15,9 +16,30 @@ import (
 	"github.com/DefangLabs/defang/src/pkg/types"
 	defangv1 "github.com/DefangLabs/defang/src/protos/io/defang/v1"
 	"github.com/bufbuild/connect-go"
+	compose "github.com/compose-spec/compose-go/v2/types"
 	"github.com/spf13/cobra"
 )
 
+var managed_resources = []string{"x-defang-redis", "x-defang-postgres"}
+
+func IsOnlyManagedResources(project *compose.Project) bool {
```

Comment:
I think this function returns true if there are _any_ managed services, not if there are _only_ managed services.

Reply:
it's a little bit of odd logic but the jist is that we return false as soon as there's a service without any extensions or if there are extension but at least one is not part of the managed resources list.

It only returns true after not finding an early exit.